### PR TITLE
Adds logs surrounding the preparation and launch of asynchronous operations

### PR DESCRIPTION
**Pull request by flamingbear**
**Summary:**

Adds logging around the launch of asynchronous operations.

Addresses: Integration test failure investigation.

## Changes

* Throws a ton of logs at the code that is invoked to start our asynchronous operations lambda.

## Additions

With this code, I rebuilt the async_operation Docker container and pushed up v39 to the cumulus sandbox account.

~I configured Bamboo SECRET_ASYNC_OPERTION_IMAGE with the new v39 for my branch only.~
(This didn't affect the build, I needed to fix the async_operation_image_version 432c3c3)

## Questions

- [ ] Is the README.md file at api/ecs/async_operation/README.md still correct? I did not (and cannot) push anything to the cumuluss [sic] repo.
- [ ] can we discuss what needs to be done to actually release this? (which repos does the docker image need to get to and how does it get there?)
- [ ] what do we have to do for SIT?

## PR Checklist

- [X ] Update CHANGELOG
- [n/a] Unit tests
- [X] Ad-hoc testing - Deploy changes and test manually
- [n/a] Integration tests

Ad-hoc testing included running async operations from the dashboard and
verifying the code was invoked correctly as well as the new expected logs were
generated.

### CODE REVIEW

**Comment by flamingbear** [comment]
@Jkovarik requested you for your expertise and since we already talked about this.  super low priority unless you begin to see failures with the async integration tests as I'm out all week.

**Comment by Jkovarik** [comment]
Responding to questions: 

> Is the README.md file at api/ecs/async_operation/README.md still correct? I did not (and cannot) push anything to the cumuluss [sic] repo.

Yes, but incomplete.   We're pushing updates to ECR and targeting that in the `example` project (https://github.com/nasa/cumulus/blob/master/example/cumulus-tf/main.tf#L95) , however the default on cumulus module `async_operation_image`  variable is using the cumuluss variable.

In short, we need to publish cumuluss for users and update the default *and* update our ECR.

**Comment by Jkovarik** [comment]
> can we discuss what needs to be done to actually release this? (which repos does the docker image need to get to and how does it get there?)

Yes, let's pair Monday

**Comment by Jkovarik** [comment]
> what do we have to do for SIT?

Update SIT's ECR with the updated image

File: example/cumulus-tf/variables.tf [hunk 1]
```diff
@@ -338,7 +338,7 @@ variable "rds_admin_access_secret_arn" {
 variable "async_operation_image_version" {
   description = "docker image version to use for Cumulus async operations tasks"
   type = string
-  default = "36"
+  default = "39"
```
Comment (Jkovarik) [inline example/cumulus-tf/variables.tf:341]: As noted in the comments, this needs to be pushed to ECR for our tests to work, and if we want users to use 39, we'll need to push it to cumuluss and update the default in the`cumulus` module.

File: CHANGELOG.md [hunk 2]
```diff
@@ -20,6 +20,9 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
 
 ### Changed
 
+- **CUMULUS-NONE**
+  - Adds logging to ecs/async-operation Docker conatiner that launches async
```
Comment (Jkovarik) [inline CHANGELOG.md:24]: I'm not sure this is right, the updates are to the async-operations repo. 

maybe 'Updates default async operation image version to 39 to take advantage of (link to changelog) logging updates'?

**Review by Jkovarik — CHANGES_REQUESTED** [review]
Minor feedback, changes for CL and pending pairing next week.

File: CHANGELOG.md (hunk 2 above)
Comment (Jkovarik) [reply CHANGELOG.md:24]: Wihdrawn.    😓

**Review by Jkovarik — CHANGES_REQUESTED** [review]
This all looks fine, change request is just pending pairing on release.

**Review by Jkovarik — APPROVED** [review]

**Pull request merged**